README.md

```diff
@@ -47,19 +47,25 @@ More detailed build instructions can be found [here](docs/install_vortex.md).
 - [Yosys](https://github.com/YosysHQ/yosys)
 - [Sv2v](https://github.com/zachjs/sv2v)
 ### Install development tools
-    $ sudo apt-get install build-essential
-    $ sudo apt-get install binutils
-    $ sudo apt-get install python
-    $ sudo apt-get install uuid-dev
-    $ sudo apt-get install git
+    $ sudo apt-get install build-essential git binutils python3 uuid-dev wget ocl-icd-opencl-dev
 ### Install Vortex codebase
-    $ git clone --depth=1 --recursive https://github.com/vortexgpgpu/vortex.git
-    $ cd Vortex
+    $ git clone https://github.com/vortexgpgpu/vortex.git
+    $ cd vortex
+    $ git submodule sync
+    $ git submodule update --init --recursive --jobs 6
```

Comment:
Did this lead to a significant performance improvement?
I think the user can decide themselves if they want to parallelize and the copy-paste default should be non parallelized.

```suggestion
    $ git submodule update --init --recursive
```

Reply:
I found that it did but yeah this is optional.
I guess a lot of users dont know this option exists/are too lazy to add it in (like me)